skyportal/facility_apis/ztf.py

```diff
@@ -0,0 +1,230 @@
+from requests import Request, Session
+from datetime import datetime, timedelta
+from astropy.time import Time
+import urllib
+
+from . import FollowUpAPI
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+
+if not cfg['app.ztf.port'] is None:
+    ZTF_URL = f"{cfg['app.ztf.protocol']}://{cfg['app.ztf.host']}:{cfg['app.ztf.port']}"
+else:
+    ZTF_URL = f"{cfg['app.ztf.protocol']}://{cfg['app.ztf.host']}"
```

Comment:
A bit more idiomatic, otherwise sounds like I not can haz cheeseburger? 
```suggestion
if cfg['app.ztf.port'] is None:
    ZTF_URL = f"{cfg['app.ztf.protocol']}://{cfg['app.ztf.host']}"
else:
    ZTF_URL = f"{cfg['app.ztf.protocol']}://{cfg['app.ztf.host']}:{cfg['app.ztf.port']}"
```